src/baseCommand.ts

```diff
@@ -49,6 +50,17 @@ export abstract class BaseCommand extends Command {
         /**
          * Configuration
          */
+
+        // This happen when providing network through env var
+        // see Oclif PR: https://github.com/oclif/parser/pull/64
+        // will be removed if PR is accepted
+        if (!UTILS.getNetworksList().includes(flags.network)) {
+            throw new Error(`Expected --network=${
```

Comment:
Not the same output: 
with native check: 
```
 ›   Error: Expected --network=lkmlk to be one of: devnet, local
 ›   See more help with --help
```

with your check:
```
Error: Expected --network=dalinet to be one of: [devnet|local]
            See more help with --help
    at StatusCommand.run (~/workspaces/uns-cli/src/baseCommand.ts:58:19)
    at StatusCommand._run (~/workspaces/uns-cli/node_modules/@oclif/command/lib/command.js:44:31)
```

Reply:
See FlagInvalidOptionError : https://github.com/oclif/parser/blob/6577b9ecc9f34d7a78b46f17fa2176d90f5a9244/src/errors.ts#L84